CONTRIBUTING.md

```diff
@@ -183,11 +183,24 @@ We generally support the usage of functional programming and lambda, usage of it
 ### References
 For passing any parameter which isn't a primitive prefer to use references/const references to pass them into functions or other places as a copy can be avoided that way.  
 In addition, always use a const reference rather than a normal reference unless the argument needs to be modified in-place as the compiler knows the intent far better in that case.
+However in constructors if you are copying to a member variable `std::move` is preferred as it allows copy-elision in some circumstances.
 ```cpp
 void DoSomething(const Class& class, u32 primitive);
-void ClassConstructor(const Class& class) : class(class) {} // Make a copy directly from a `const reference` for class member initialization
+void ClassConstructor(Class class) : class(std::move(class)) {} // Make a copy directly from a `const reference` for class member initialization
 ```
 
+### Member Variable Shadowing
+Shadowing of class member variables should be avoided aside from in constructors when they are exclusively used in the initialisation list.
```

Comment:
We tend to use the American `initialization` rather than `initialisation`, there's an instance of it directly above this

Reply:
Smh you and your inferior spelling